src/CMakeLists.txt

```diff
@@ -187,6 +183,10 @@ if(ENABLE_ASEPRITE_EXE)
   add_executable(${main_target}
     main/main.cpp
     ${main_resources})
+
+  if(ENABLE_DESKTOP_INTEGRATION)
+    add_subdirectory(desktop)
+  endif()
```

Comment:
It looks like this was moved here to use `add_dependencies(${main_target} ${extension_target})` but the main target (the `aseprite` binary) doesn't depend on the thumbnailer. The Aseprite.app depends on the thumbnailer, but our cmake files don't build the .app bundle (that's why the Info.plist/entitlements files are not included in the repo).

I'll include these files (Info.plist/entitlements) in the repo, and probably we could make a target to build the Aseprite.app (so users can build their local .app bundle), but this dependency/change shouldn't be required (the thumbnailer is built explicitly with `ninja AsepriteThumbnailer.qlgenerator` currently)